app/presenters/activity_presenter.rb

```diff
@@ -109,6 +109,15 @@ def flow_with_code
     "#{flow} (#{to_model.flow})"
   end
 
+  def sustainable_development_goals
+    goals = [sdg_1, sdg_2, sdg_3].compact
+    return if goals.blank?
+
+    goals.map { |goal|
+      I18n.t("form.label.activity.sdg_options.#{goal}")
+    }.join(" / ")
+  end
```

Comment:
I wonder if it might be better to display these as a bulleted (or numbered) list to better reflect the fact that the order of them matters